public/app/plugins/datasource/graphite/state/context.tsx

```diff
@@ -50,13 +50,13 @@ export const GraphiteQueryEditorContext = ({
     if (state) {
       dispatch(actions.queriesChanged(queries));
     }
-  }, [dispatch, queries]);
+  }, [dispatch, queries, state]);
```

Comment:
I'm guessing these changes are unrelated and causing linting errors?

Reply:
Yes exactly, they just showed up as typescript warnings, so I thought I will just fix them. :)